backend/app/Http/Controllers/UserController.php

```diff
@@ -34,30 +33,36 @@ public function create(Request $request)
             'token' => $token,
         ];
 
-
-        return response()->json($response, 201);
+        return response()->json($response);
     }
 
     public function update(Request $request)
     {
         $field = $request->validate([
-            'email' => ['required'],
+            'name' => ['required'],
             'password' => ['required'],
         ]);
-        $res = User::where('email', $request->email)->first();
-        if (!$res || !Hash::check($field['password'], $res->password)) {
+        $res = User::where('name', $request->name)->first();
+        if ($res) {
+            if (!Hash::check($field['password'], $res->password)) {
```

Comment:
```suggestion
            if ($res && !Hash::check($field['password'], $res->password)) {
```

Reply:
Hello sir, regarding on this one. I needed the code to return specifically if the the username exists, and had it checked with the retrieved password it $res returns true. Should I just replace it with a "Wrong Credentials" as a whole?